CHANGELOG.md

```diff
@@ -24,6 +24,8 @@ _No breaking changes!_
 **Project enhancements:**
 
 - Updated official test coverage to support Ruby 3.3 [[#335](https://github.com/panorama-ed/memo_wise/pull/335)]
+- Added `alt_memery` and `memoist3` to benchmarks [#339](https://github.com/panorama-ed/memo_wise/pull/339)
+  - Updated benchmark results in `README.md` to Ruby 3.3.5
```

Comment:
Nitpick: please make this a top-level item with its own PR link (even though it's the same):

```markdown
- Updated benchmark results in `README.md` to Ruby 3.3.5 [[#339](https://github.com/panorama-ed/memo_wise/pull/339)]
```